source/User_Guide/email_activity_beta.md

```diff
@@ -10,7 +10,7 @@ seo:
   description:
 ---
 
-The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Enhanced email activity and storage gives you increased visibility into the email delivery flow to help expedite the troubleshooting process.  
+The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Our latest enhancementss to Email Activity give you increased visibility into the email delivery flow to help expedite the troubleshooting process.
```

Comment:
Fix "enhancementss" please. :)

Reply:
Also, can you rephrase this in a way that doesn't make it sound like a release note?